views/qunitrunner.mustache

```diff
@@ -9,11 +9,7 @@
 {{#styles}}<link rel="stylesheet" href="{{.}}">{{/styles}}
 </head>
 <body>
-<h1 id="qunit-header">Tests</h1>
-<h2 id="qunit-banner"></h2>
-<div id="qunit-testrunner-toolbar"></div>
-<h2 id="qunit-userAgent"></h2>
-<ol id="qunit-tests"></ol>
+<div id="qunit"></div>
```

Comment:
as long as you use `<div id="qunit"></div>` as proposed, QUnit 1.17.0 won't break anything on PhantomJS. Anyway, it's worth waiting 1 day to get 1.17.1 fixing this minor regression.